autoreject/autoreject.py

```diff
@@ -12,8 +12,8 @@
 from mne.utils import logger, ProgressBar
 
 from sklearn.base import BaseEstimator
-from sklearn.grid_search import RandomizedSearchCV
-from sklearn.cross_validation import KFold
+from sklearn.model_selection import RandomizedSearchCV
+from sklearn.model_selection import KFold
```

Comment:
Humm... As I thought this would be a good idea as autoreject is also in a dev version currently. scikit-learn 0.18 is long overdue and will be released very shortly... Possibly before next month end I believe...